backend/de.metas.externalsystem/src/main/java/de/metas/externalsystem/leichmehl/ExternalSystemLeichMehlConfig.java

```diff
@@ -55,12 +51,6 @@ public class ExternalSystemLeichMehlConfig implements IExternalSystemChildConfig
 
 	boolean pluFileExportAuditEnabled;
 
-	@NonNull
-	List<ExternalSystemLeichMehlConfigProductMapping> productMappings;
```

Comment:
i assume we are removing the productMappings from here because they are too many.... or?

Reply:
Moved to product-tab and not a child config of externalSystems anymore